src/styles/components/_doc-item.scss

```diff
@@ -38,19 +38,27 @@
       }
     }
 
-    .end {
-      @media (max-width: 996px) {
-        display: none;
-      }
-    }
-
     > .container {
       > .row {
         > .col {
           display: flex;
           justify-content: center;
 
           min-width: 0;
+
+          // Docs Demo
+          &.col--4 {
+            @media (max-width: 1364px) {
+              display: none;
+            }
+          }
+
+          // Table of contents
+          &.col--3 {
+            @media (max-width: 996px) {
+              display: none;
+            }
```

Comment:
Can we add comments describing what this does and why it is necessary?

Reply:
Removed the `col--3` since I noticed that it's being removed completely under `max-width` so it doesn't need to be hidden via CSS. I did add a comment for `col--4` since it can be removed by JS.